toolsrc/src/vcpkg.cpp

```diff
@@ -227,24 +227,47 @@ int main(const int argc, const char* const* const argv)
 #endif
 
     VcpkgCmdArguments args = VcpkgCmdArguments::create_from_command_line(fs, argc, argv);
+    if (const auto p = args.debug.get()) Debug::g_debugging = *p;
     args.imbue_from_environment();
     args.check_feature_flag_consistency();
 
-    if (const auto p = args.disable_metrics.get()) Metrics::g_metrics.lock()->set_disabled(*p);
-    if (const auto p = args.print_metrics.get()) Metrics::g_metrics.lock()->set_print_metrics(*p);
-    if (const auto p = args.send_metrics.get()) Metrics::g_metrics.lock()->set_send_metrics(*p);
-    if (const auto p = args.debug.get()) Debug::g_debugging = *p;
-
-    if (args.send_metrics.has_value() && !Metrics::g_metrics.lock()->metrics_enabled())
     {
-        System::print2(System::Color::warning,
-                       "Warning: passed either --sendmetrics or --no-sendmetrics, but metrics are disabled.\n");
-    }
-    if (args.print_metrics.has_value() && !Metrics::g_metrics.lock()->metrics_enabled())
-    {
-        System::print2(System::Color::warning,
-                       "Warning: passed either --printmetrics or --no-printmetrics, but metrics are disabled.\n");
-    }
+        auto metrics = Metrics::g_metrics.lock();
+        if (const auto p = args.disable_metrics.get())
+        {
+            metrics->set_disabled(*p);
+        }
+
+        auto disable_metrics_tag_file_path =
+            System::get_exe_path_of_current_process().replace_filename(fs::u8path(".disable-metrics"));
```

Comment:
I imagine this should be: 
```suggestion
            System::get_exe_path_of_current_process().replace_filename(fs::u8path("vcpkg.disable-metrics"));
```

Reply:
🤦‍♂️